pkgs/applications/networking/browsers/w3m/default.nix

```diff
@@ -1,16 +1,18 @@
 { stdenv, fetchurl
-, sslSupport ? true
-, graphicsSupport ? false
-, mouseSupport ? false
-, ncurses, openssl ? null, boehmgc, gettext, zlib
-, imlib2 ? null, xlibsWrapper ? null, fbcon ? null
-, gpm-ncurses ? null
+, ncurses, boehmgc, gettext, zlib
+, sslSupport ? true, openssl ? null
+, graphicsSupport ? true, imlib2 ? null
+, x11Support ? graphicsSupport, libX11 ? null
```

Comment:
> I say after I fix `LIB` thingy, test that it doesn't break the frame
> buffer and everyone agrees to these changes, then you should apply your
> `fetchpatch`-changes (on top or in parallel, shouldn't matter, to this).

... because this way it would be much easier to merge for both of us:
this way you'll need to throw away changes from your patches and it will
just work, while the way proposed by you'll require merging at the top
of the file.